src/transformers/trainer.py

```diff
@@ -1354,3 +1422,25 @@ def prediction_step(
         if labels is not None:
             labels = labels.detach()
         return (loss, logits.detach(), labels)
+
+    def floating_point_ops(self, model: nn.Module, inputs: Dict[str, Union[torch.Tensor, Any]]):
+        """
+        For models with a `floating_point_ops` method (e.g. models that inherit from `PretrainedModel`), uses
+        that method to compute the number of floating point operations for every backward + forward pass. If using
+        another model, either implement such a method in the model or override this method.
+
+        Args:
+            model (:obj:`nn.Module`):
```

Comment:
Can't we use `self.model`?

Reply:
yep, changed it, allows us to save a few lines in the main method too

Reply:
We can remove the docstring as well